release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
+        {
+            "channel-version": "2.1",
+            "latest": "2.1.4",
+            "release-date":"2018-09-11",
+            "lts": true,
+            "eol": false,
+            "eol-date": null,
```

Comment:
I would suggest and empty string rather than null for eol-date. This would make processing easier.

Reply:
Change will be in the next commit.